app/tools/android/customize.py

```diff
@@ -211,6 +211,31 @@ def CustomizeThemeXML(name, fullscreen, manifest):
   theme_file.close()
 
 
+# Customize for different arch
+def CustomizePerArch(app_info):
```

Comment:
I do not see any arch related in this func, maybe the function change to `CustomizeManifestByAppInfo()`